src/translations.php

```diff
@@ -3,14 +3,17 @@
 return [
     "currentStreak" => [
         "en" => "Current Streak",
+        "es" => "Racha Actual",
         "ja" => "現在のストリーク",
     ],
     "totalContributions" => [
         "en" => "Total Contributions",
+        "es" => "Número de Contribuciones",
```

Comment:
I think we can change this to make it shorter as discussed on Discord

```suggestion
        "es" => "Todas Contribuciones",
```